### PR TITLE
Update functionMap

### PR DESCRIPTION
**Pull request by zonuexe**

### CODE REVIEW

File: resources/functionMap.php [hunk 1]
```diff
@@ -12017,7 +12017,7 @@
 'stream_set_write_buffer' => ['int', 'fp'=>'resource', 'buffer'=>'int'],
 'stream_socket_accept' => ['resource|false', 'serverstream'=>'resource', 'timeout='=>'float', '&w_peername='=>'string'],
 'stream_socket_client' => ['resource|false', 'remoteaddress'=>'string', '&w_errcode='=>'int', '&w_errstring='=>'string', 'timeout='=>'float', 'flags='=>'int', 'context='=>'resource'],
-'stream_socket_enable_crypto' => ['int|bool', 'stream'=>'resource', 'enable'=>'bool', 'cryptokind='=>'int', 'sessionstream='=>'resource'],
+'stream_socket_enable_crypto' => ['0|bool', 'stream'=>'resource', 'enable'=>'bool', 'crypto_method='=>'int', 'session_stream='=>'resource'],
```
Comment (ondrejmirtes) [inline resources/functionMap.php]: Parameters don't need to be renamed

File: resources/functionMap.php [hunk 2]
```diff
@@ -3334,7 +3334,7 @@
 'getimagesize' => ['array{0:int, 1: int, 2: int, 3: string, mime: string, channels?: int, bits?: int}|false', 'imagefile'=>'string', '&w_info='=>'array'],
 'getimagesizefromstring' => ['array{0:int, 1: int, 2: int, 3: string, mime: string, channels?: int, bits?: int}|false', 'data'=>'string', '&w_info='=>'array'],
 'getlastmod' => ['int|false'],
-'getmxrr' => ['bool', 'hostname'=>'string', '&w_mxhosts'=>'array', '&w_weight='=>'array'],
+'getmxrr' => ['bool', 'hostname'=>'string', '&w_mxhosts'=>'list<string>', '&w_weight='=>'list<0, 65535>'],
```
Comment (ondrejmirtes) [inline resources/functionMap.php]: What effect should this have? The types aren't enforced: https://phpstan.org/r/25c3b187-4888-4aa4-86ad-5e350f16346f

**Review by ondrejmirtes — CHANGES_REQUESTED** [review]

File: resources/functionMap.php (hunk 2 above)
Comment (zonuexe) [reply resources/functionMap.php]: I see, it turns out that typing like `@param-out` cannot be solved by modifying only the functionMap.

https://phpstan.org/r/3c7a8a08-abce-45c9-b310-0bcb4e4b6ba4

The execution results of the function are as follows:

スクリーンショット 2023-11-27 3 30 35

**Comment by ondrejmirtes** [comment]
Thank you!

**Pull request merged**